src/api/app/views/webui/project/_breadcrumb_items.html.haml

```diff
@@ -11,7 +11,7 @@
       List of All Projects
   - elsif current_page?(new_project_path)
     %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      New
+      = @project.name.to_s.ends_with?(':') ? 'Create Subproject' : 'Create Project'
```

Comment:
Hm, no project ends with `:` right?

Reply:
Is because of the change on the [subproject  link](https://github.com/openSUSE/open-build-service/pull/9131/files#diff-3c48e54b21a58442ed416f6d8f083a3cR19)

Reply:
When the `project#new` page is loaded, `@project.name` is empty when we wish to create a project (so we clicked on one of the `Create Project` links from the homepage or the `project#index` view). However, when we wish to create a subproject, it will end with `:` since I made this change [here](https://github.com/openSUSE/open-build-service/pull/9131/files#diff-3c48e54b21a58442ed416f6d8f083a3cR19). I double-checked how it was done in the modal and it's done differently. It wouldn't be possible to know we're adding a project with [this way](https://github.com/openSUSE/open-build-service/pull/9131/files#diff-f9d333f44ffa40e2e74dd79f7c241997L10) though.

Reply:
Please have another look, I've changed this by using a namespace. This was already in place, I improved it slightly by having a consistent naming.